Open Judge System/Web/OJS.Web/Areas/Contests/Views/Results/_FullPartial.cshtml

```diff
@@ -0,0 +1,131 @@
+                            @problemResult.BestSubmission.SubmissionType
+                            <br />
+                            if (!problemResult.BestSubmission.IsCompiledSuccessfully)
+                            {
+                                @:Compilation failed!
+                            }
+
+                            foreach (var testRun in problemResult.BestSubmission.TestRuns)
+                            {
+                                var style = testRun.IsZeroTest ? "-ms-opacity: 0.3; opacity: 0.3" : string.Empty;
+                                switch (testRun.ResultType)
+                                {
+                                    case TestRunResultType.CorrectAnswer:<span style="@style" class="glyphicon glyphicon-ok text-success" title=@Resource.Answer_correct></span>break;
+                                    case TestRunResultType.WrongAnswer:<span style="@style" class="glyphicon glyphicon-remove text-danger" title=@Resource.Answer_incorrect></span>break;
+                                    case TestRunResultType.TimeLimit:<span style="@style" class="glyphicon glyphicon-time text-danger" title=@Resource.Time_limit></span>break;
+                                    case TestRunResultType.MemoryLimit:<span style="@style" class="glyphicon glyphicon-hdd text-danger" title=@Resource.Memory_limit></span>break;
+                                    case TestRunResultType.RunTimeError:<span style="@style" class="glyphicon glyphicon-asterisk text-danger" title=@Resource.Runtime_error></span>break;
+                                }
+                            }
+                        }
+                        else
+                        {
+                            <span>@Resource.No_solution</span>
+                        }
+
+                        </td>
+                    }
+
+                    <td>@participant.Total</td>
+                </tr>
+            }
+
+        </table>
+                    }
+                    else
+                    {
+                        <h2>@Resource.NoMoreResults</h2>
+                    }
+
+    @if (results.PageCount > 1)
+    {
+        @Html.PagedListPager(
+            results,
+            page => Url.Action<ResultsController>(
+                c => c.FullPartial(
+                    Model.Id,
+                    isOfficial,
+                    page,
+                    results.PageSize)),
+            PagedListRenderOptions.EnableUnobtrusiveAjaxReplacing(new AjaxOptions
+            {
+                HttpMethod = "GET",
+                UpdateTargetId = ResultsContainerId,
+                LoadingElementId = LoadingMaskContainerId,
+                OnSuccess = OnPageChangedEventHandlerName
+            }))
+    }
+    
+    @Html.Partial("_LoadingMaskPartial", LoadingMaskContainerId)
```

Comment:
No need for the "Partial" part in the name, the "\_" already implicitly tells us it's a partial view.

Reply:
Done in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/223